### PR TITLE
feat: added Home Page

### PR DESCRIPTION
**Pull request by srv-pranjal**
# Home Page

## Description

 - Added backend for videos and categories
 - Added Navbar and Sidebar
 - added display of videos and filter by categories
 - added copy to clipboard feature
 
 
image

Filter by category
image

Deploy Preview: https://fino-world-git-home-srv-pranjal.vercel.app/

### CODE REVIEW

File: src/components/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { NavLink } from "react-router-dom";
+import "./Sidebar.css";
+import { BiHomeCircle, BiLike } from "react-icons/bi";
+import { MdOutlineWatchLater } from "react-icons/md";
+import { RiPlayList2Fill, RiHistoryFill } from "react-icons/ri";
+import { useSidebar } from "contexts";
+import classNames from "classnames";
```
Comment (TejasShekar) [inline src/components/Sidebar/Sidebar.js:7]: Interesting package to combine classnames conditionally

**Review by TejasShekar — APPROVED** [review]
Clean code. Very good folder structure.
Liked your utility functions as well.

File: src/components/CategoryChip/CategoryChip.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.chip {
+  border: 0.1rem solid var(--white-color);
```
Comment (VishnuVNairCodes) [inline src/components/CategoryChip/CategoryChip.css:2]: Consider converting length units less than 1 rem to pixels

**Comment by VishnuVNairCodes** [comment]
Good to merge. Consider converting length units less than 1 rem to pixels

**Review by VishnuVNairCodes — APPROVED** [review]

**Pull request merged**